next.config.js

```diff
@@ -21,7 +21,8 @@ module.exports = {
       'a.storyblok.com',
       'images.ctfassets.net',
       'images.prismic.io',
-      'cdn.aglty.io'
+      'cdn.aglty.io',
+      'localhost'
```

Comment:
Should localhost be here?

Reply:
Because Strapi is self hosted we need next/image to allow images to display during development as well. I don't think there's another way to handle that unfortunately!